nebula_common/include/nebula_common/continental/continental_ars548.hpp

```diff
@@ -68,6 +69,7 @@ inline std::ostream & operator<<(
   os << "Host Port: " << arg.configuration_host_port << '\n';
   os << "Sensor Port: " << arg.configuration_sensor_port << '\n';
   os << "UseSensor Time: " << arg.use_sensor_time << '\n';
+  os << "RadarInfoRateSubsample: " << arg.radar_info_rate_subsample << '\n';
```

Comment:
Please add spaces to make it more human-readable (the `UseSensor Time` above as well if possible)

Reply:
Addressed in https://github.com/tier4/nebula/pull/284/commits/a6e7a7b256bcf42f65152cca964db3655122c246